src/v2/Apps/Settings/Routes/MyCollection/Components/SettingsMyCollectionArtworks.tsx

```diff
@@ -28,6 +34,10 @@ const SettingsMyCollectionArtworks: FC<SettingsMyCollectionArtworksProps> = ({
   relay,
 }) => {
   const [loading, setLoading] = useState(false)
+  const [hasDismissedMessage, setHasDismissedMessage] = useState(
+    window.localStorage.getItem("HAS_SEEN_MY_COLLECTION_MESSAGE_BANNER") ===
```

Comment:
Ah, is this working? Since this relies on `window` I would expect this to break during the SSR pass. Is this component only mounted on the client via a `QueryRenderer`?

Reply:
@damassi it works on dev. I am not sure if this component is only mounted via a QueryRenderer. I should move this logic into a hook that only runs client, like useEffect 🤔

Reply:
Interesting -- i would investigate this and be sure. If things are executing within a query renderer it should be fine (since query renderers only execute on the client) but not that this code will break if things ever get moved outside of a query renderer into our formal server-side-rendering flow. 

If it works though, no reason to move to a hook! We should just be sure to understand why it's working.

Reply:
Following recent changes and refactors, I encountered the expected window not available error. Now using it in useEffect instead.

Reply:
Sounds good! Lets keep an eye on UX flashing in an unstable way (server renders, then this banner suddely appears on client-side mount), and if so we can return to this and perhaps use a cookie instead. Nothing blocking here, though 👍